lib/gzip_vas.c

```diff
@@ -167,12 +167,18 @@ int nx_function_end(nx_devp_t nxhandle)
 {
 	int rc = 0;
 
-	rc = munmap(nxhandle->paste_addr - 0x400, 4096);
-	if (rc < 0) {
-		fprintf(stderr, "munmap() failed, errno %d\n", errno);
-		return rc;
-	}
-	close(nxhandle->fd); /* see issue 164 comment */
+		rc = munmap(nxhandle->paste_addr - 0x400, 4096);
+		if (rc < 0) {
+			fprintf(stderr, "munmap() failed, errno %d\n", errno);
+			return rc;
+		}
+
+		rc = close(nxhandle->fd); /* see issue 164 comment */
+		if (rc < 0) {
+			fprintf(stderr, "close() failed, errno %d\n", errno);
+			return rc;
+		}
```

Comment:
Wrong indentation.

Reply:
I had actually wrapped these calls on if statements to check if paste_addr and fd are not invalid, to make the function idempotent. But looks like the ifs got lost during a rebase. Fixed on the next push.